apps/judicial-system/backend/src/app/modules/case/case.service.ts

```diff
@@ -1384,6 +1414,30 @@ export class CaseService {
       }
     }
 
+    if (
+      isIndictment &&
+      [CaseState.SUBMITTED, CaseState.RECEIVED].includes(updatedCase.state)
+    ) {
+      const judgeChanged =
```

Comment:
I like it better like this. I think 

`if (judgeChanged)` 

is more readable than 

`if (isJudgeChanged)`

I could swing either way tho. If you have strong opinions, I'll change it ;)

Reply:
Nah just a nit for consistency given the above: `isIndictment`, where we start with the verb and then the noun. But I don't think we follow that as a strict pattern for boolean vars, right?

My only issue with the above (or in general when not starting with a verb) is that it could read as an object representing the judge that was changed, if that makes sense? But this is not a blocker so I will resolve ☺️

Reply:
I can see how having `is` in front of boolean values is beneficial. Will change. Thanks 🙏